modules/spam/text.py

```diff
@@ -375,7 +370,10 @@ def brainfuck(self, update: Update, context: CallbackContext) -> None:
                 update.message.reply_text(f"This is taking too much time :/")
                 return
 
-        update.message.reply_text("Interpreted:")
-        message = context.dispatcher.bot.send_message(update.message.chat_id, result)
-        message.from_user = update.message.from_user
-        context.update_queue.put(Update(0, message))
+        if not result:
+            update.message.reply_text("No output generated.\nCongratulations, you warmed up the planet for nothing.")
+        else:
+            update.message.reply_text("Output:")
+            message = context.dispatcher.bot.send_message(update.message.chat_id, result)

```

Comment:
Maybe something like

```suggestion
            message = context.dispatcher.bot.send_message(update.message.chat_id, result, reply_to_message_id=update.message.id)
```

?

Reply:
sure, then i'll change it a bit